include/logging/log.h

```diff
@@ -424,8 +424,7 @@ static inline char *log_strdup(const char *str)
  *
  */
 #define LOG_LEVEL_SET(level) \
-	static const u32_t __log_level __attribute__((unused)) = \
-			_LOG_LEVEL_RESOLVE(level)
```

Comment:
`_LOG_LEVEL_RESOLVE` was not needed here. It was copy pasted from macros which had optional log level and were using default if nothing provided. In that case argument `level` is obligatory